src/components/task/Task.tsx

```diff
@@ -8,69 +8,62 @@ import {
 import './index.css';
 import {
   Card,
-  Flex,
-  FlexItem,
+  Badge,
+  GalleryItem,
+  TextContent,
 } from '@patternfly/react-core';
 import {OkIcon, DownloadIcon} from '@patternfly/react-icons';
-
+import {CardHead, CardHeader, CardFooter, CardBody, CardActions} from '@patternfly/react-core';
+import imgAvatar from '../assets/logo/imgAvatar.png';
 export interface TaskPropObject {
```

Comment:
I don't see any difference between TaskPropObject and Mockdata? Are both the interfaces representing the same object?